tests/unit/test_get_request_url.py

```diff
@@ -0,0 +1,27 @@
+import os
+import pytest
+
+from typing import Callable, Optional
+
+from mz_bokeh_package.utilities import Environment
+
+staging_url = 'https://api-staging.materials.zone/v1beta1' + '/'
+production_url = 'https://api.materials.zone/v1beta1' + '/'
+custom_api_host_url = 'http://staging.materials.zone:5000' + '/'
+
+# The order of the tests is important. set_env_to_development_with_own_host sets the environment variable
```

Comment:
@captKuk I think this comment has to be updated, no? Cause I don't see a function called `set_env_to_development_with_own_host`. Better yet, I think it's nicer to remove the comment and simply clean the API_HOST environment variable in the beginning of the test using:
```
if 'API_HOST' in os.environ:
        del os.environ['API_HOST']
```
Then the order doesn't matter anymore.